src/test/kotlin/uk/gov/dluhc/registercheckerapi/mapper/RegisterCheckResultMapperTest.kt

```diff
@@ -81,9 +83,7 @@ internal class RegisterCheckResultMapperTest {
             val actual = mapper.fromRegisterCheckResultRequestApiToDto(queryParamRequestId, apiRequest)
 
             // Then
-            assertThat(actual).usingRecursiveComparison().isEqualTo(expected)
-            verify(instantMapper).toInstant(createdAt)
-            verify(instantMapper).toInstant(applicationCreatedAt)
```

Comment:
Adding the new fields caused an additional call to `instantMapper` which breaks these verifications that `instantMapper` is being called a certain number of times with these particular arguments.

Rather than tweaking these verifys to account for the additional call, I just deleted them because I can't see what they're adding to this test apart from making it brittle.

We ignore `historicalSeachEarliestDate` since it was making the test very brittle if the seconds ticked over while the test was running. See Lewis's slack message https://softwire.slack.com/archives/C05KAM40NUU/p1707229862731319

Reply:
cool that all makes sense, surprised the field wasn't already excluded, but possibly caught why later on in the assertion test helper file